### PR TITLE
Forms v4 visual refresh

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
Changes:
- totally reworked admin and block editor UIs
- cleaned up some parts
- improved UI and UX

Protip: go commit by commit 😄

image

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

**Review by piqusy — CHANGES_REQUESTED** [review]
Wow, the improvement is gigantic, great job overall 🚀 
I haven't checked everything, "only" checked 100 files 😄 

I left some suggestions in the comments and please fix linter errors.

Also I've notice you are using `px`, `rem` & `em` values intermittently in the admin area.
Is there a reason why you couldn't use a same unit? The 1px is for borders and such is fine IMO

P.S. I see some commits were made while I was reviewing, please ignore what I commented on if it is already fixed

File: src/Blocks/components/admin-settings-section/admin-settings-section-admin.scss [hunk 1]
```diff
@@ -202,100 +137,97 @@
...
 		svg {
-			margin-right: 1rem;
 			width: 1.5rem;
 			height: 1.5rem;
 
 			transition: {
-				property: transform;
+				property: transform, color;
 				timing-function: ease-in-out;
 				duration: 0.3s;
```
[the remark this answers is not in the document]
Comment (goranalkovic-infinum) [reply src/Blocks/components/admin-settings-section/admin-settings-section-admin.scss:178]: Nope, it should use the first one for all of 'em.

File: src/Blocks/components/card/card-admin.scss [hunk 2]
```diff
@@ -1,70 +1,253 @@
...
+				grid-area: title;
+			}
+
+			#{$this}__subtitle {
+				grid-area: subtitle;
+			}
+
+			#{$this}__links {
+				grid-area: links;
+				gap: 0.5ch;
```
[the remark this answers is not in the document]
Comment (goranalkovic-infinum) [reply src/Blocks/components/card/card-admin.scss:98]: All intented 😄

File: src/Blocks/components/conditional-tags/components/conditional-tags-forms-options.js [hunk 3]
```diff
@@ -102,96 +109,108 @@ export const ConditionalTagsFormsOptions = (attributes) => {
...
+								newData.splice(index, 1);
+								setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: newData });
+								return null;
+							}
+
+							return (
+								<div key={index} className='es-h-spaced'>
+									<ConditionalTagsItem index={index} />
+
+									<Button
```
[the remark this answers is not in the document]
Comment (goranalkovic-infinum) [reply src/Blocks/components/conditional-tags/components/conditional-tags-forms-options.js:168]: A bit of visual separation, whitespace doesn't cost anything 😛

**Review by iruzevic — APPROVED** [review]

File: src/Blocks/components/submit/submit-admin.scss [hunk 4]
```diff
@@ -1,20 +1,69 @@
 @import './submit-style.scss';
 
+a.es-submit,
 button.es-submit {
```
Comment (piqusy) [inline src/Blocks/components/submit/submit-admin.scss:4]: Could you nest `.es-submit`

File: src/Blocks/components/submit/submit-editor.scss [hunk 5]
```diff
@@ -1 +1,6 @@
 @import './submit-style.scss';
+
+a.es-submit,
+button.es-submit {
```
Comment (piqusy) [inline src/Blocks/components/submit/submit-editor.scss:4]: Please nest `.es-submit`

File: src/Blocks/components/submit/submit-style.scss [hunk 6]
```diff
@@ -1,3 +1,4 @@
+a.es-submit,
 button.es-submit {
```
Comment (piqusy) [inline src/Blocks/components/submit/submit-style.scss:2]: Same as above

**Review by piqusy — CHANGES_REQUESTED** [review]
Looking good, left some minor suggestions for some of the files I haven't had a change to review the last time😊

**Review by piqusy — APPROVED** [review]
Approved ... Haven't checked all the files tho' 😄

**Pull request merged**